WordPress/Classes/System/ReaderPresenter.swift

```diff
@@ -143,7 +143,11 @@ final class ReaderPresenter: NSObject, SplitViewDisplayable {
         case .addTag:
             let addTagVC = UIHostingController(rootView: ReaderTagsAddTagView())
             addTagVC.modalPresentationStyle = .formSheet
-            addTagVC.preferredContentSize = CGSize(width: 420, height: 124)
+            let preferredHeight: CGFloat = 124
+            addTagVC.sheetPresentationController?.detents = [.custom(resolver: { _ in
+                preferredHeight
+            })]
```

Comment:
You may want to add an "is iPhone" check here because I believe this view will be displayed as a 124-tall sheet on iPad, too.

Reply:
It looks good on iPad.

<img width="958" alt="Screenshot 2024-10-29 at 9 03 21 PM" src="https://github.com/user-attachments/assets/464ef099-40f1-45a9-a300-4ecfc1f4f27a">